web/backend/src/Server.ts

```diff
@@ -461,13 +475,18 @@ function sendToDela(dataStr: string, req: express.Request, res: express.Response
 
 // Secure /api/evoting to admins and operators
 app.use('/api/evoting/*', (req, res, next) => {
-  if (!isAuthorized(['admin', 'operator'], req)) {
-    console.log('role is:', req.session.role);
-    res.status(400).send('Unauthorized - only admins and operators allowed');
-    return;
-  }
-
-  next();
+  e.then((enforcer) => {
```

Comment:
is there are a reason not to use `isAuthorized` here?

Reply:
I just forget it. It is fixed !